db/migrate/20230728114353_create_users.rb

```diff
@@ -0,0 +1,10 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :username
```

Comment:
db level indexes missing username should have index and be unique for faster searching